packages/types/models.ts

```diff
@@ -1,5 +1,14 @@
 import mongoose from 'mongoose'
 
+export const REPORT_STATUS = {
+  NOMINATION: 'Nomination',
+  REVIEW: 'Review',
+  REPORT: 'Report',
+  COMPLETE: 'Complete',
+} as const
+
+export type ReportStatus = (typeof REPORT_STATUS)[keyof typeof REPORT_STATUS]
+
```

Comment:
this is a really useful technique! I got it from [here](https://youtu.be/6M9aZzm-kEc?t=277)